recipes/entrypoints/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp entrypoints.py $SP_DIR
```

Comment:
That would make sense. Yeah, I was trying to look for some sort of description, but must have missed it.